### ISSUE TITLE
Migration to 0.19

### ISSUE THREAD

**Issue by noiseOnTheNet** [rundis/elm-bootstrap#126]
Hi @rundis, I really like your package and would like to see it in 0.19; is there a way to contribute?

**Comment by nullbio** [comment]
Hi @rundis. I'm starting a new app and was planning on using 0.19. Just wondering if there is any concrete plans or timelines on a migration for elm-bootstrap?

**Comment by rundis** [comment]
TLDR; I'm working on it. Have some patience. 0.19 is just out.

- The package has dependencies to other packages that haven't been ported yet. Fortunately not many, as I've gotten burned in the past. Most notably the testing story is not complete yet (html test)
- My editor (Light Table) which imho has had excellent deep Elm integration was rendered useless by the 0.19 upgrade so I've been trying out alternatives until I have time to update the its plugin.
- I also need to port the doc site (elm-bootstrap.info). 
- Gotta decide whether and how to support 0.18 going forward 

Right now more hands isn't going to help (most likely the opposite, me spending time communicating, explaining and coordinating rather than doing work) . I'll let me people know if and when it makes sense to 
help out.

In terms of timeline, it's ready when it's ready. No promises, but I'm thinking weeks rather than months.

There's now a new [0.19 branch](https://github.com/rundis/elm-bootstrap/tree/elm-019) which contains porting efforts so far.

**Comment by aarondl** [comment]
Hi there @rundis. Eagerly awaiting the 0.19 version. How is it going? Is there anything you need help with at this juncture? Happy to pitch in if things have changed in regards to needing more hands.

**Comment by jbzar** [comment]
Hello @rundis. Are you still working on this one? I am also starting a project in Elm -- would like to use Elm-bootstrap. Does the 0.19 branch work yet? Do you need help?

**Comment by rundis** [comment]
elm-bootstrap 5.0.0 out now. So closing this. I'm bound to have missed something, so please report issues/problems you encounter (and ideally specify that its related to 0.19.X version of Elm). Not planning on supporting elm-bootstrap for elm 0.18 going forward.

**Comment by nullbio** [comment]
Thank you @rundis -- much appreciated!
